ffw/ButtonsRPC.js

```diff
@@ -179,44 +179,72 @@ FFW.Buttons = FFW.RPCObserver.create(
           break;
         }
         case 'Buttons.SubscribeButton': {
-          try {
-              const params = request.params;
-              const appID = params.appID
-              const resultCode = this.subscribeButton(appID, params.buttonName);
-              console.log("Button " + params.buttonName + " " + resultCode + " resultCode");
+          const buttonName = request.params.buttonName;
+          const appID = request.params.appID;
+          if(buttonName.includes("NAV_")) {
+            this.navButtonSubscriptionToggle(appID, buttonName, true);
+            const resultCode = SDL.SDLModel.data.resultCode.SUCCESS;
+            console.log("Button " + buttonName + " " + resultCode + " resultCode");
```

Comment:
Did you forget to remove console.log?

Reply:
@YuriiYefimenko No.This is for testing purposes. It's OK